common/cuda_hip/base/dev_lapack_bindings.hpp

```diff
@@ -0,0 +1,18 @@
+// SPDX-FileCopyrightText: 2017 - 2025 The Ginkgo authors
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifndef GKO_COMMON_CUDA_HIP_BASE_DEV_LAPACK_BINDINGS_HPP_
+#define GKO_COMMON_CUDA_HIP_BASE_DEV_LAPACK_BINDINGS_HPP_
+
+
+#if defined(GKO_COMPILING_CUDA)
+#include "cuda/base/cusolver_bindings.hpp"
+#elif defined(GKO_COMPILING_HIP)
+#include "hip/base/hipsolver_bindings.hip.hpp"
```

Comment:
This PR seems to handle cuda only, will you also add the hipsolver here?

Reply:
Yes, the plan was to add hipSOLVER bindings here once the interface is finalized.

Reply:
I added hipSOLVER in 5e6dc8e3bb955a7a29bd97e37781af4bdc4bc21b.